apiserver/getcachedvhdimages.go

```diff
@@ -0,0 +1,41 @@
+package apiserver
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+
+	agent "github.com/Azure/agentbaker/pkg/agent"
+)
+
+const (
+	// RoutePathGetCachedVHDImages the route path to get cached vhd images.
+	RoutePathGetCachedVHDImages string = "/getcachedvhdimages"
+)
+
+// GetCachedVHDImages endpoint for getting cached VHD images.
+func (api *APIServer) GetCachedVHDImages(w http.ResponseWriter, r *http.Request) {
```

Comment:
might also rename to make it clear these are container images we're talking about